freqtrade/templates/sample_hyperopt_advanced.py

```diff
@@ -136,56 +136,59 @@ def sell_indicator_space() -> List[Dimension]:
             Categorical([True, False], name='sell-fastd-enabled'),
             Categorical([True, False], name='sell-adx-enabled'),
             Categorical([True, False], name='sell-rsi-enabled'),
-            Categorical(['sell-bb_upper',
+            Categorical(['sell-boll',
                          'sell-macd_cross_signal',
-                         'sell-sar_reversal'], name='sell-trigger')
+                         'sell-sar_reversal'],
+                        name='sell-trigger')
         ]
 
     @staticmethod
     def sell_strategy_generator(params: Dict[str, Any]) -> Callable:
         """
         Define the sell strategy parameters to be used by hyperopt
         """
-        def populate_sell_trend(dataframe: DataFrame, metadata: dict) -> DataFrame:
+        def populate_exit_trend(dataframe: DataFrame, metadata: dict) -> DataFrame:
             """
             Sell strategy Hyperopt will build and use
             """
             # print(params)
-            conditions = []
+            exit_long_conditions = []
             # GUARDS AND TRENDS
             if 'sell-mfi-enabled' in params and params['sell-mfi-enabled']:
-                conditions.append(dataframe['mfi'] > params['sell-mfi-value'])
+                exit_long_conditions.append(dataframe['mfi'] > params['sell-mfi-value'])
             if 'sell-fastd-enabled' in params and params['sell-fastd-enabled']:
-                conditions.append(dataframe['fastd'] > params['sell-fastd-value'])
+                exit_long_conditions.append(dataframe['fastd'] > params['sell-fastd-value'])
             if 'sell-adx-enabled' in params and params['sell-adx-enabled']:
-                conditions.append(dataframe['adx'] < params['sell-adx-value'])
+                exit_long_conditions.append(dataframe['adx'] < params['sell-adx-value'])
             if 'sell-rsi-enabled' in params and params['sell-rsi-enabled']:
-                conditions.append(dataframe['rsi'] > params['sell-rsi-value'])
+                exit_long_conditions.append(dataframe['rsi'] > params['sell-rsi-value'])
 
```

Comment:
Honestly, i'd not touch this part.

It's the legacy interface - which will eventually be removed - so while we can align method names - i'd not change the content of the methods.

Reply:
Not touch just the sample_hyperopt_advanced file?

Reply:
Just let me know which files are legacy and that I shouldn't touch

Reply:
I'd assume all files in `tests/strategy/strats` (create copies of either default_strategy or hyperoptable_strategy with the new approach), as well as default_hyperopt (again, create a copy - potentially naming the copy "legacy_hyperopt" though :laughing: )